src/main/java/ie/ait/bteam/drcare/data/model/User.java

```diff
@@ -36,13 +36,14 @@
 	@Getter
 	@Setter
 	@Column(unique = true)
+	@NotNull
 	private String email;
 
 	@Getter
 	@Setter
 	@Column
 	@NotNull
-	private Boolean isAdmin;
+	private Boolean isAdmin = false;
```

Comment:
boolean by default is false. if you need to do any of this, do on DTOs

Reply:
Object Boolean is null by default while primitive is false... That's why I initialised this to false. Will change it now tho.